### ISSUE TITLE
Show updater stuck

### ISSUE THREAD

**Issue by Guesch** [SickChill/SickChill#6045]
### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: Master
OS: Synology
Browser:
What you did:
What happened: The automatic show updater is stuck in the middle of processing shows for more than 30h. Unfortunately it happens on a semi regular basis, but untilI notice it my shows are not being updated at all (for time for months :( )
What you expected: For the queue to clean and the shows to be updated automatically 
Logs: Couldn't find any ;(

Note: If it matters I'm behind a VPN not allowing port forwarding, but everything works fine beside that (except for the check update process unable to reach github)
```
PASTE LOGS HERE
```
![image](https://user-images.githubusercontent.com/61683661/75658796-2f1d6b00-5c69-11ea-85c5-69b8d31b0d49.png)

**Comment by ronrosenfeld** [comment]
I am having the same problem.  In addition, if I go to a show on the list, there is a `This show is queued and awaiting an update. ` message.  On your Synology, are you running in Docker or in the native OS?  I am running in Docker.

**Comment by Guesch** [comment]
Native OS (my DS414 doesn't support Docker unfortunately)

**Comment by ronrosenfeld** [comment]
Yes, but the fact that it is happening both within and without Docker makes me think that Docker is not the problem.

**Comment by nydave69** [comment]
This show is queued and awaiting an update.
A lot of my shows show this banner...   OS: Windows 7
Things seem to be working and I don't see any errors in the log.
Not sure where to look for the updater queue....

Found the queue under server status.....  Has to be over a hundred shows listed as Force Update or Refresh going back 3 days...

**Comment by BenjV** [comment]
@Guesch 
To my knowledge the screenshot you are showing is not a part of SickChill.
So most likely you are not using the SickChill.

Advice is to remove it and install SickChill.

**Comment by ronrosenfeld** [comment]
@BenjV Check "Server Status"

**Comment by BenjV** [comment]
Found it.
But that queue is always empty on my system.

**Comment by Guesch** [comment]
That would actually the bug we are reporting: Our doesn't clean up as it should ;)

**Comment by ronrosenfeld** [comment]
@BenjV  Mine empties if I restart Sickchill (as happens when it gets updated, and then repopulates at about 4AM Local time.  This morning, though, the queue type is "Refresh", and the number of items in the queue is only 11.  I'll see what it looks like tomorrow.

**Comment by ronrosenfeld** [comment]
This morning, the queue seems to be "stuck" in the same place, and additional files have been added to the list, some of which are of type `FORCE-UPDATE`

**Comment by gcampionpae** [comment]
I've also been seeing the same thing with my SickChill.  Most of my shows aren't updating with new dates that episodes are coming out etc.  I've had my debug log on and did a restart this evening and one show I know that hasn't updated is The Walking Dead for example.  I did a force update on Avenue 5 and that will work properly but the daily update of all my shows seems to be really hit or miss.  I've attached the entire log from today if it helps.  Thanks for any help!

[sickchill.log](https://github.com/SickChill/SickChill/files/4314699/sickchill.log)

**Comment by miigotu** [comment]
Update and try again, new api key for thetvdb

**Comment by ronrosenfeld** [comment]
@miigotu  Don't know about the latest update  (Commit: f9161da71a75dbabf83fd862bc4e46a0b4391352 ) this morning (which I will check tomorrow AM), but the same problem is continuing. The list gets longer every morning, and the first item in the list remains the same.  It still empties out upon restarting the program.

**Comment by ronrosenfeld** [comment]
@miigotu  This is the first morning in a long time that there have not been shows "stuck in the queue". I will keep monitoring this.

**Comment by ronrosenfeld** [comment]
I wrote too soon.  The Show Queue under Server Status again shows most of the shows lined up there, with Queue type being either Refresh or Force Update, and no progress over the past hour.

**Comment by Guesch** [comment]
@ronrosenfeld It is potentially totally unrelated, but I think I managed to fix my issue by deleting the cache.db (and as I was there I deleted the failed.db as well).
It has been 10 days ish and so far so good.

Having said that, when I tried to enable my VPN through OpenVPN (I'm using NordVPN who do not allow port forwarding), the process failed as it couldn't connect to TvDB. But for the first time ever I actually had an error in the log telling me so. 

Long story short: Removed VPN (which sucks) and cleared the cache seems to have made the trick for my system.
Hope it helps!

@miigotu : Allowing user to clear their cache from the GUI would be neat (not the image etc cache but the cache.db!)

Cheers,

**Comment by BenjV** [comment]
I also use NordVPN with OpenVPN and do not have any problem connecting to Tvdb, so your problem must be something else.
And connect from SickChill to Tvdb has nothing to do with portforwarding, that is only of any importance when you want to connect from outside to your network not from Sickchill to Tvdb.

Of course Tvdb has lots of issues lately so now and then there is a problem connecting but that's an tvdb issue.

**Comment by ronrosenfeld** [comment]
My Show Queue has now been clear for the last two mornings.  Fingers crossed.

**Comment by ronrosenfeld** [comment]
@miigotu 

Stuck again,  Since `2020-03-24 04:29:24` with `The Good Doctor` again first in the queue.  And it looks like every show is listed.  With Queue Type being either `Force Update` or `Refresh` 

And multiple issues **this morning** of the type:

````
2020-03-25 04:29:36 INFO     SHOWUPDATER :: Automatic update failed: Show_Name is already being updated by Post-processor or manually started, can't update again until it's done.
````

It seems that there all ran:

```
2020-03-20 04:28:24 INFO     SHOWUPDATER :: ShowUpdater for tvdb Api V3 starting
2020-03-20 04:28:25 INFO     SHOWUPDATER :: Last update: 2020-03-19 04:10:21
2020-03-21 04:28:38 INFO     SHOWUPDATER :: ShowUpdater for tvdb Api V3 starting
2020-03-21 04:28:39 INFO     SHOWUPDATER :: Last update: 2020-03-20 04:28:25
2020-03-22 04:28:52 INFO     SHOWUPDATER :: ShowUpdater for tvdb Api V3 starting
2020-03-22 04:28:53 INFO     SHOWUPDATER :: Last update: 2020-03-21 04:28:39
2020-03-23 04:29:07 INFO     SHOWUPDATER :: ShowUpdater for tvdb Api V3 starting
2020-03-23 04:29:07 INFO     SHOWUPDATER :: Last update: 2020-03-22 04:28:53
2020-03-24 04:29:18 INFO     SHOWUPDATER :: ShowUpdater for tvdb Api V3 starting
2020-03-24 04:29:22 INFO     SHOWUPDATER :: Last update: 2020-03-23 04:29:07
```

After the last one, things got stuck and I now see:

```
2020-03-25 04:29:33 INFO     SHOWUPDATER :: ShowUpdater for tvdb Api V3 starting
2020-03-25 04:29:35 INFO     SHOWUPDATER :: Last update: 2020-03-24 04:29:22
2020-03-25 04:29:36 INFO     SHOWUPDATER :: Automatic update failed: Series_Name is already being updated by Post-processor or manually started, can't update again until it's done.
2020-03-25 04:29:36 INFO     SHOWUPDATER :: Automatic update failed: Series_Name is in process of being updated by Post-processor or manually started, can't update again until it's done.
2020-03-25 04:29:36 INFO     SHOWUPDATER :: Automatic update failed: Series_Name is in process of being updated by Post-processor or manually started, can't update again until it's done.

… (Multiple repeats for every Show)
```
Any thoughts?

I can clear the queue by rebooting, but it seems it gets stuck after a while.

**Comment by Guesch** [comment]
UNfortunately the issue re-appeared for me.. :( Anyway we can "re-open" the bug as perhaps it was closed a bit too quickly? :(
